google_api_availability_platform_interface/lib/src/implementations/method_channel_google_api_availability.dart

```diff
@@ -0,0 +1,77 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import '../google_api_availability_platform_interface.dart';
+import '../models/google_play_services_availability.dart';
+
+/// An implementation of [GoogleApiAvailabilityPlatform] that uses method channels.
+class MethodChannelGoogleApiAvailability extends GoogleApiAvailabilityPlatform {
```

Comment:
Adding a default implementation is no longer recommended, meaning this file should be deleted. See also comments in the `google_api_availability_platform_interface.dart` file.